src/bci_build/package/appcontainers.py

```diff
@@ -807,3 +808,82 @@ def _get_nginx_kwargs(os_version: OsVersion):
     )
     for tomcat_major, os_version in product(_TOMCAT_VERSIONS, ALL_BASE_OS_VERSIONS)
 ]
+
+SPACK_CONTAINERS = [
+    ApplicationStackContainer(
+        name="spack",
+        package_name="spack-image",
+        pretty_name="Spack package manager for supercomputers - build container",
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        other_tags=["next"]
+        if not (
+            (os_version in CAN_BE_LATEST_OS_VERSION)
+            or (os_version in RELEASED_OS_VERSIONS)
+        )
+        else None,
+        version="%%spack_version%%",
+        version_in_uid=False,
+        replacements_via_service=[
+            Replacement(
+                regex_in_build_description="%%spack_version%%",
+                package_name="spack",
+                parse_version="patch",
+            ),
+            Replacement(
+                regex_in_build_description="%%spack_version%%",
+                file_name="README.md",
+                package_name="spack",
+                parse_version="patch",
+            ),
+        ],
+        package_list=[
+            "spack",
+            "autoconf",
+            "bison",
+            "ccache",
```

Comment:
I don't think that having plain ccache in a container makes a lot of sense. The ccache cache will be put into a folder in the container's ephemeral file system and all you'll be getting is a bigger and slower container. Unless spack specifically needs ccache, I'd opt to remove it.
```suggestion
```

Reply:
Yeah, we should probably leave it out. 
It might bring an advantage when building multiple containers  but it will require to mount a more persistent cache directory into the container.
Moreover, Spack wraps calls to the compiler itself, so Spack can be configured to use ccache. Then it will do so without the need for additional wrapping. Users interested in using it may want to create a derived container.